runtime/cere/src/lib.rs

```diff
@@ -1486,38 +1491,15 @@ pub type UncheckedExtrinsic =
 pub type SignedPayload = generic::SignedPayload<RuntimeCall, SignedExtra>;
 /// Extrinsic type that has already been checked.
 pub type CheckedExtrinsic = generic::CheckedExtrinsic<AccountId, RuntimeCall, SignedExtra>;
-
-pub struct NominationPoolsMigrationV4OldPallet;
-impl Get<Perbill> for NominationPoolsMigrationV4OldPallet {
-	fn get() -> Perbill {
-		Perbill::zero()
-	}
-}
-/// Migrations that set `StorageVersion`s we missed to set.
```

Comment:
it seems that this was missed and added, and now it removed again - why?

Reply:
1. We executed `session::ClearOldSessionStorage` migration in update to [v0.9.42](https://github.com/Cerebellum-Network/blockchain-node/pull/262/files#diff-d544a22153b5b835b7371624a808d65d3a76f05a465a1d80a9ec759822dfeb3c)
2. But `session::ClearOldSessionStorage` was missing a storage version update.
3. When we realized it we added a fix `SetStorageVersions`.
4. Now `SetStorageVersions` is executed and we can remove it.